Assets/Scripts/Props/HeavyLift.cs

```diff
@@ -25,11 +25,11 @@ protected override void OnMouseDown()
             {
                 _liftAnimation.LiftBoulderAnims();
             }
+            
         }
 
         protected override void OnMouseUp()
         {
-            EnableCollisions(true);
```

Comment:
Oh yeah this is dumb huh. Oops.

Reply:
Hehehe

Reply:
I feel like we already had this and some change you made must have gotten lost, idk

Reply:
Nah it was only the log that I made better

Reply:
Oh I take it all back.